Samples/auth/Office-Add-in-SSO-NAA/README.md

```diff
@@ -63,7 +63,7 @@ For more information on how to register your application, see [Register an appli
 
 1. Clone or download this repository.
 1. From the command line, or a terminal window, go to the root folder of this sample at `/samples/auth/Office-Add-in-SSO-NAA`.
-1. Open the `src/taskpane/authConfig.ts` file.
+1. Open the `src/taskpane/msalconfig.ts` file.
 1. Replace the placeholder "Enter_the_Application_Id_Here" with the Application ID that you copied.
 1. Save the file.
```

Comment:
Line 77 says "This will start the web server and sideload the add-in to Excel". Right now it's configured for Word so just need to change this to Word. also please add a note on how to change to Excel or PowerPoint (by editing the start command in package.json).